native/src/console/site.rs

```diff
@@ -50,6 +47,11 @@ impl Run for SiteLaunchCommand {
             bail!("Runtime not installed");
         }
 
+        if storage.config.hash != components::runtime::b3hasher() && !storage.config.hash.is_empty()
```

Comment:
Well, does it not work when just using file modification dates? Because that would probably be a bit easier and you wouldn't need to store hash (and add another dependency).

Reply:
Sure, I guess that hashes are more reliable than modification dates. But I would still make it calculate both hashes instead of storing them.